src/ui/app/components/transaction.jsx

```diff
@@ -495,9 +495,17 @@ const getTimestamp = (date) => {
 };
 
 const getAddressCredentials = (address) => {
-  const cmlAddress = Loader.Cardano.Address.from_bech32(address);
-  return [cmlAddress.payment_cred()?.to_cbor_hex(), cmlAddress.staking_cred()?.to_cbor_hex()];
-}
+  try {
+    const cmlAddress = Loader.Cardano.Address.from_bech32(address);
+    return [
+      cmlAddress.payment_cred()?.to_cbor_hex(),
+      cmlAddress.staking_cred()?.to_cbor_hex(),
```

Comment:
So that it's consistend and not 'null | undefined | ...'
```suggestion
      cmlAddress.payment_cred()?.to_cbor_hex() || null,
      cmlAddress.staking_cred()?.to_cbor_hex() || null,
```